### PR TITLE
add custom play.html and preview_pane.html templates

### PR DESCRIPTION
**Pull request by dvoong**

### CODE REVIEW

**Comment by dvoong** [comment]
couple fo things to consider,

- code styling is not complete in the query text area, django sql explorer uses the code mirror library to style the sql
  - reimplement it from scratch - pros, could just be a few lines
  - rip it out from django sql explorer, pros, could just be a copy paste, cons, could come with a lot of additional bloat that is difficult to debug
- ordering functionality not included in the sql query preview
- buttons after the query text area, in the original django-sql-explorer these are buttons with additional dropdown options, I've flattened out the hierarchy because I haven't seen any examples of these in govuk sites

**Comment by dvoong** [comment]
all button functionality added except the `Format` button

**Comment by dvoong** [comment]
need to add hide schema functionality

File: core/templates/explorer/preview_pane.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	  {% for h in headers %}
+	  <th class="govuk-table__header">{{ h }}</th>
+	  {% endfor %}
+	</tr>
+
+	<tbody class="govuk-table__body">
+	  {% if data %}
+	  {% for row in data %}
+	  <tr class="govuk-table__row">
+	    {% for i in row %}
```
Comment (marcuspp) [inline core/templates/explorer/preview_pane.html:31]: give `i` a more descriptive name

File: core/templates/explorer/preview_pane.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      {% else %}
+      Showing {{ rows }} rows from a total of {{ total_rows }}
+      {% endif %}
+    </p>
+
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+	<tr class="govuk-table__row">
+	  {% for h in headers %}
+	  <th class="govuk-table__header">{{ h }}</th>
```
Comment (marcuspp) [inline core/templates/explorer/preview_pane.html:23]: give `h` a more descriptive name

File: core/templates/explorer/play.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+			<span class="govuk-visually-hidden">Error:</span> {{ error|escape }}
+		    </span>
+		    {% endif %}
+
+		    {% if form.connections|length > 1 %}
+		    <div class="govuk-form-group">
+			<label class="govuk-label" for="id_connection">
+			    Connection
+			</label>
+			{{ form.connection }}
```
Comment (abbas123456) [inline core/templates/explorer/play.html:34]: I don't think this is enough to render the dropdown correctly, please look at https://github.com/uktrade/data-explorer/blob/develop/core/templates/explorer/query.html#L77-L93

File: static/js/index.js [hunk 4]
```diff
@@ -3,3 +3,72 @@ import VueRx from 'vue-rx'
...
+	    if(element.getAttribute('schemaHidden') === null) {
+		element.setAttribute('schemaHidden', true);
+	    	element.setAttribute(
+		    'src',
+		    '../schema/' + document.getElementById('id_connection').value
+		);
+	    }
+
+	    let schemaHidden = element.getAttribute('schemaHidden');
+	    console.log(`schemaHidden: ${schemaHidden}`);
```
Comment (marcuspp) [inline static/js/index.js:38]: Can the console log statements be removed?

File: core/templates/explorer/play.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
+{% extends 'core/base.html' %}
+
+{% load static %}
+
+{% load i18n %}
```
Comment (marcuspp) [inline core/templates/explorer/play.html:5]: This is not required

File: core/templates/explorer/play.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+	    <iframe src="about:blank" height="828px" frameBorder="0" id="schema_frame"></iframe>
+	</div>
+
+    </div>
+    
+</div>
+
+{% include 'explorer/preview_pane.html' %}
+
+{% load render_bundle from webpack_loader %}
```
Comment (marcuspp) [inline core/templates/explorer/play.html:112]: This should go in the base file with load at the top of the template

Comment (dvoong) [reply core/templates/explorer/play.html:112]: I think you mean the `load render_bundle...` bit? The `include 'explorer/preview_pane.html` should be in that template

**Comment by abbas123456** [comment]
You need to add validation to the SQL text area, submitting a blank one results in `which results in syntax error at end of input LINE 1: DECLARE cur_bcab18b291 CURSOR WITH HOLD FOR ^`

File: core/templates/explorer/play.html (hunk 6 above)
Comment (marcuspp) [reply core/templates/explorer/play.html]: The load and render bundle bit not the include

**Pull request merged**